src/phpFastCache/Drivers/memcache.php

```diff
@@ -47,8 +47,8 @@ public function __construct($config = array())
         if (class_exists('Memcache')) {
             $this->instant = new MemcacheSoftware();
 
-            if (isset($config[ 'compress_data' ]) &&  $config[ 'compress_data' ] === true) {
-                $this->flags = MEMCACHE_COMPRESSED;
+            if (array_key_exists('compress_data', $config) &&  $config[ 'compress_data' ] === true) {
```

Comment:
This is in no way an improvement, the second condition will now be evaluated even  if the value is `null`, whereas `isset` returns false for null-values.